people/benefits.md

```diff
@@ -67,19 +71,19 @@ The conference allowance accounts for all the days you are away at the conferenc
 
 Conferences count as working days (7.5h per day). Transport to conferences counts too (up to 7.5h per day).
 
-The conference bonus requires that you write a report about the conference when you get back. It can be published on Intra or on any of our blogs. The conference fee amount is reimbursed using your allowance.
+The conference bonus requires that you write a report about the conference when you get back. It can be published on Intra or on any of our blogs. 
 
 For example, Dejan goes to a three-day conference in London:
 
     * Conference fee is $500.
-    * Flights, trains and taxis are $300.
-    * Hotel and food are $400.
-
-    Dejan charges the company $1200 in allowances so his conference allowance is calculated as:
+    * Flights, trains and taxis are $500.
+    * Hotel and food are $600.
+    
+    The conference allowance is calculated as (with a days travel each way):
 
-    * $1500 (5 * $200/day + $500 for the conference fee)
+    * $1500 (5 * $200/day)
 
-    Before the trip, Dejan's allowance was $500 in credit and after the trip, it is now $800.
+    Therefore Dejan charges the company, $500 in business expenses and $1100 in allowances. And before the trip, Dejan's allowance was $500 in credit so after the trip, it is now $900.
```

Comment:
`Therefore Dejan charges the company, $500` to `Therefore Dejan charges the company $500` (removed comma).

Reply:
Therefore is followed by a comma when used at the beginning of the sentence. `Therefore, Dejan charges the company $500...`

Reply:
`Therefore Dejan charges` or `Therefore, Dejan charges` both work I think. I would prefer the one without the comma because I wouldn't make a pause between `Therefore` and `Dejan` when speaking.

I think it's a [more modern style to start dropping some of the commas](https://english.stackexchange.com/a/26501), and more formal/traditional to leave commas in. Both cases are OK.

Reply:
@suzil The comma (could even be semi-colon) after `company` signifies the list of items that follow.